envoy/formatter/http_specific_formatter.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
```

Comment:
This is a strange file name `http_specific_formatter.h` for a file which simply declares a class named `HttpFormatterContext`. Should it perhaps be named `http_formatter_context.h`?

Reply:
got it.